api/client/api_alert.go

```diff
@@ -126,8 +126,8 @@ func (a *AlertApiService) AlertsTeamsGet(ctx context.Context) ([]string, *http.R
 
 /*
 AlertApiService Lists alerts for given model.
- * @param ctx context.Context - for authentication, logging, cancellation, deadlines, tracing, etc. Passed from http.Request or context.Background().
- * @param modelId
+  - @param ctx context.Context - for authentication, logging, cancellation, deadlines, tracing, etc. Passed from http.Request or context.Background().
```

Comment:
Why is this `*` removed from the auto-generated code now? Is it simply because of a newer version of `swagger-codegen` ?

Reply:
`gofmt` did that and not swagger